src/UpgradeRule/PHP/Visitor/RenameClassesVisitor.php

```diff
@@ -71,10 +146,42 @@ protected function handleStringUpdate(Scalar\String_ $stringNode)
                 return;
             }
 
-            // Substitute MyClass::class literal in place of string
             $baseName = $this->logUseStatement($replacement);
+
+            // Show warning if this replacement may be invalid
+            if (isset($this->renameWarnings[$baseName]) && $this->showPrompt) {
```

Comment:
Console output in black'n'white is a bit unpleasant.
![image](https://user-images.githubusercontent.com/1168676/51504357-6ce47900-1e45-11e9-8c5e-7b30d62e48d7.png)

Most Windows users won't get the colours. If there's a good way of saying I'm replacing the instance of the word at column X in the line Y, it would be great. But otherwise, I'm not overly fuss about it.